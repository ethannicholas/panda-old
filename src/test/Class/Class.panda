var s := "Hello"
var c:Class? := s.class
while c != null {
    Console.writeLine(c.name)
    c := c.superclass
}

Console.writeLine("Success")

