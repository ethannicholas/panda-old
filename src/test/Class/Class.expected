panda.core.String
panda.core.Value
panda.core.Immutable
panda.core.Object
Success
